<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PositionSystem</class>
 <widget class="QMainWindow" name="PositionSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PositionSystem</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1271</width>
      <height>811</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QTabBar::tab{ min-height: 60px; min-width:60px;}</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Home">
     <attribute name="title">
      <string>HomePage</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="GPS">
     <attribute name="title">
      <string>GPS Location</string>
     </attribute>
     <widget class="QPushButton" name="GPSOpenPort">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>210</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Open Port</string>
      </property>
     </widget>
     <widget class="QLabel" name="GPSLati">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>499</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Latitude</string>
      </property>
     </widget>
     <widget class="QPushButton" name="GPSSearchPort">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Search Port</string>
      </property>
     </widget>
     <widget class="QComboBox" name="GPSRatecomboBox">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>310</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1200</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="GPSHighlineEdit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>580</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="GPStextEdit">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>110</y>
        <width>481</width>
        <height>561</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="GPSHigh">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>580</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>High</string>
      </property>
     </widget>
     <widget class="QPushButton" name="GPSReceive">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>630</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Receive GPS INF</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="GPSLonglineEdit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>540</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="GPSPort">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>260</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="GPSSatNumlineEdit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>420</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="GPSTitle">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>481</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 100 30pt &quot;Times New Roman&quot;;
</string>
      </property>
      <property name="text">
       <string>GPS LOCATION AND LOG</string>
      </property>
     </widget>
     <widget class="QLabel" name="GPSLong">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>540</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Longitude</string>
      </property>
     </widget>
     <widget class="QLabel" name="GPSSatNum">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>420</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Sat Num</string>
      </property>
     </widget>
     <widget class="QLabel" name="GPSState">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>465</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>State</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="GPSLatilineEdit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>500</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="GPSPortcomboBox">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>260</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>none</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="GPSPortRate">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>310</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Port Rate</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="GPSStatelineEdit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>460</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="GPSLogcheckBox">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>418</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>LOG</string>
      </property>
     </widget>
     <widget class="QPushButton" name="GPSSentCommand">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>350</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Sent GPS Command</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="GPSdateTimeEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Acq">
     <attribute name="title">
      <string>Acquisition</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Upd">
     <attribute name="title">
      <string>Update TLE </string>
     </attribute>
     <widget class="QDateTimeEdit" name="UpdateTimeEdit">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>60</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
     </widget>
     <widget class="QWebEngineView" name="TLEVIew" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>541</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="UpdateTLE">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>60</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="IridiumNext">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>60</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Update Iridium NEXT TLE from Network</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="UptextEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>541</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Iridium">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Update Iridium TLE from Network</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="UptextEdit_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>100</y>
        <width>491</width>
        <height>571</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="Uplabel">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 100 30pt &quot;Times New Roman&quot;;
</string>
      </property>
      <property name="text">
       <string>TLE UPDATE TOOL</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Loc">
     <attribute name="title">
      <string>Location</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1085</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPosition_System_based_on_Iridium">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAcqiration"/>
    <addaction name="actionGPS_Locotion"/>
    <addaction name="actionTLEupdation"/>
    <addaction name="actionLocation"/>
   </widget>
   <widget class="QMenu" name="menuCNTLab">
    <property name="title">
     <string>About</string>
    </property>
    <widget class="QMenu" name="menuCreater">
     <property name="title">
      <string>Creater</string>
     </property>
     <addaction name="actionsunguiyu96_gmail_com_2"/>
     <addaction name="actionhttps_github_com_BIOSmode"/>
    </widget>
    <addaction name="actionVersion"/>
    <addaction name="menuCreater"/>
   </widget>
   <addaction name="menuPosition_System_based_on_Iridium"/>
   <addaction name="menuCNTLab"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAcqiration">
   <property name="text">
    <string>Acqiration</string>
   </property>
  </action>
  <action name="actionTLEupdation">
   <property name="text">
    <string>TLEupdation</string>
   </property>
  </action>
  <action name="actionLocation">
   <property name="text">
    <string>Location</string>
   </property>
  </action>
  <action name="actionsunguiyu96_gmail_com">
   <property name="text">
    <string>sunguiyu96@gmail.com</string>
   </property>
  </action>
  <action name="actionCNT">
   <property name="text">
    <string>CNTLAB</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>Version</string>
   </property>
  </action>
  <action name="actionsunguiyu96_gmail_com_2">
   <property name="text">
    <string>sunguiyu96@gmail.com</string>
   </property>
  </action>
  <action name="actionhttps_github_com_BIOSmode">
   <property name="text">
    <string>https://github.com/BIOSmode</string>
   </property>
  </action>
  <action name="actionGPS_Locotion">
   <property name="text">
    <string>GPS Locotion</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
